btree/btree.c

```diff
@@ -2864,7 +2865,12 @@ static int ff_keysize(const struct nd *node)
 
 static int ff_valsize(const struct nd *node)
```

Comment:
Make this an inline function. We should also convert any other few line functions to inline if possible but in a different PR.

Reply:
done.